code/game/objects/effects/forcefields.dm

```diff
@@ -29,12 +29,37 @@
 
 ///////////Mimewalls///////////
 
-/obj/effect/forcefield/mime
+/obj/structure/forcefield
+	name = "ain't supposed to see this"
+	desc = "file a github report if you do!"
+	icon = 'icons/effects/effects.dmi'
+	density = TRUE
+	anchored = TRUE
+	var/blocks_atmos = TRUE
+
+/obj/structure/forcefield/Initialize(mapload)
+	. = ..()
+	if(blocks_atmos)
+		air_update_turf(TRUE)
+
+/obj/structure/forcefield/Destroy()
+	var/turf/turf_to_update = get_turf(src)
+	. = ..()
+	if(blocks_atmos)
+		blocks_atmos = FALSE
+		turf_to_update.air_update_turf(TRUE)
+
+/obj/structure/forcefield/CanAtmosPass(turf/T)
+	return !blocks_atmos
+
+/obj/structure/forcefield/mime
 	icon_state = "empty"
```

Comment:
a nasty side effect of this not actually having an icon is that you need to alt-click the tile to actually access this in melee. I don't know if this really can be changed though unfortunately.

Reply:
You might be able to make an icon with the minimum possible alpha to alleviate this.